### ISSUE TITLE
CAD release?

### ISSUE THREAD

**Issue by teookie** [jrlomas/miro-extruder#1]
Looks awesome. Are you planning to release the CAD by chance?

**Comment by shoeys-for-newey** [comment]
FYI to all reading this, use the Sherpa Mini's CAD release in your designs. Extruder mounting holes, filament path, and stepper motor mounting holes are all identical. I have a SV08 toolhead in the works using Miro (obviously also compatible with Sherpa Mini), release on Printables coming in a month or so. Search for "Scope Creep Industries"

**Comment by ChaosBlades** [comment]
I don't think it is going to be released. License includes No Derivatives. So if CAD was released technically you would not legally be allowed to modify it.

**Comment by Contomo** [comment]
what a shame... pretty and functional extruder, but not much use if im not even allowed to integrate it into a toolhead.

**Comment by shoeys-for-newey** [comment]
@Contomo just use the sherpa mini STEP release, dimensions are almost identical, that's what I've been doing
